Source/WebCore/platform/graphics/gstreamer/MediaPlayerPrivateGStreamer.h

```diff
@@ -169,6 +169,7 @@ class MediaPlayerPrivateGStreamer : public MediaPlayerPrivateGStreamerBase {
     void purgeInvalidTextTracks(Vector<String> validTrackIds);
 #endif
     virtual bool doSeek(const MediaTime& position, float rate, GstSeekFlags seekType);
+    void maybeFinishSeek();
```

Comment:
I know the code will technically work if this declaration remains like this, but for coherency with updateStates() and asyncChangeDone(), it makes sense to declare this method as virtual here and as override in MediaPlayerPrivateGStreamerMSE.

Reply:
sure, done